libavformat/movenc.c

```diff
@@ -3148,6 +3148,10 @@ static int mov_write_trak_tag(AVFormatContext *s, AVIOContext *pb, MOVMuxContext
     if (track->tref_tag)
         mov_write_tref_tag(pb, track);
 
+    if (track->cenc.encryption_scheme != MOV_ENC_NONE) {
+        ff_mov_cenc_write_senc_tag(&track->cenc, pb, 0);
```

Comment:
Can you document what type of boxes (or tags) are needed for cenc and cbcs? Which boxes are needed for cenc and which one for cbcs?

Reply:
The atoms are the same for cenc and cbcs. There may be cases where you can omit them if there is nothing to signal.